Packs/Lumu/Integrations/Lumu/Lumu.py

```diff
@@ -0,0 +1,1080 @@
+) -> CommandResults:
+    label_id = args.get("label_id")
+
+    response = client.retrieve_a_specific_label_request(label_id)
+
+    command_results = CommandResults(
+        outputs_prefix="Lumu.RetrieveASpecificLabel",
+        outputs_key_field="",
+        outputs=response,
+        raw_response=response,
+    )
+
+    return command_results
+
+
+def retrieve_incidents_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    page = args.get("page")
+    items = args.get("items")
+    fromdate = args.get("fromdate")
+    todate = args.get("todate")
+    status = args.get("status")
+    adversary_types = args.get("adversary-types")
+    labels = args.get("labels")
+
+    if status and isinstance(status, str):
+        status = status.split(",")
+    if adversary_types and isinstance(adversary_types, str):
+        adversary_types = adversary_types.split(",")
+    if labels and isinstance(labels, str):
+        labels = [int(label) for label in labels.split(",")]
+
+    response = client.retrieve_incidents_request(
+        page, items, fromdate, todate, status, adversary_types, labels
+    )
+    command_results = CommandResults(
+        outputs_prefix="Lumu.RetrieveIncidents",
+        outputs_key_field="",
+        outputs=response,
+        raw_response=response,
+    )
+
+    return command_results
+
+
+def retrieve_a_specific_incident_details_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    lumu_incident_id = args.get("lumu_incident_id")
+
+    response = client.retrieve_a_specific_incident_details_request(lumu_incident_id)
+    command_results = CommandResults(
+        outputs_prefix="Lumu.RetrieveASpecificIncidentDetails",
+        outputs_key_field="",
+        outputs=response,
+        raw_response=response,
+    )
+
+    return command_results
+
+
+def retrieve_a_specific_incident_context_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    lumu_incident_id = args.get("lumu_incident_id")
+    hash = args.get("hash")
+
+    response = client.retrieve_a_specific_incident_context_request(
+        lumu_incident_id, hash
+    )
+    command_results = CommandResults(
+        outputs_prefix="Lumu.RetrieveASpecificIncidentContext",
+        outputs_key_field="",
+        outputs=response,
+        raw_response=response,
+    )
+
+    return command_results
+
+
+def comment_a_specific_incident_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    lumu_incident_id = args.get("lumu_incident_id")
+    comment = generate_hmac_sha256_msg(
+        client.api_key, add_prefix_to_comment(args.get("comment"))
+    )
+    try:
+
+        response = client.comment_a_specific_incident_request(lumu_incident_id, comment)
+    except DemistoException as err:
+
+        error_msg = "Failed to parse json object from response: b''"
+        if str(err) == error_msg:
+            response = {"statusCode": 200}
+        else:
+            raise DemistoException(err)
```

Comment:
please explain this part

Reply:
mute, unmute, close, comment and mark as a read action return an empy body , so, when the built-in BaseClient try to deserializate with .json() method it show and JSONERROR but the execution was successful because statuscode was 200, 
What I meant was, a successful response return and empty body and statuscode set to 200, then, we have to control the deserialization error to continue.

https://docs.lumu.io/portal/en/kb/articles/core-concepts-api#Mute_incident___Available_for_Insights
```
curl -v --location --request POST 'https://defender.lumu.io/api/incidents/6eddaf40-938c-11ed-b0f8-a7e340234a4e/mute?key=xxxxxxxxxxxxxxxxxxxxxxxxx' --header 'Content-Type: application/json' --data-raw '{
   "comment": "mute"
}'
Note: Unnecessary use of -X or --request, POST is already inferred.
*   Trying 13.35.116.40:443...
* TCP_NODELAY set
* Connected to defender.lumu.io (13.35.116.40) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/certs/ca-certificates.crt
  CApath: /etc/ssl/certs
* TLSv1.3 (OUT), TLS handshake, Client hello (1):
* TLSv1.3 (IN), TLS handshake, Server hello (2):
* TLSv1.3 (IN), TLS handshake, Encrypted Extensions (8):
* TLSv1.3 (IN), TLS handshake, Certificate (11):
* TLSv1.3 (IN), TLS handshake, CERT verify (15):
* TLSv1.3 (IN), TLS handshake, Finished (20):
* TLSv1.3 (OUT), TLS change cipher, Change cipher spec (1):
* TLSv1.3 (OUT), TLS handshake, Finished (20):
* SSL connection using TLSv1.3 / TLS_AES_128_GCM_SHA256
* ALPN, server accepted to use h2
* Server certificate:
*  subject: CN=*.lumu.io
*  start date: Mar 18 16:01:57 2022 GMT
*  expire date: Apr 19 16:01:57 2023 GMT
*  subjectAltName: host "defender.lumu.io" matched cert's "*.lumu.io"
*  issuer: C=US; ST=Arizona; L=Scottsdale; O=GoDaddy.com, Inc.; OU=http://certs.godaddy.com/repository/; CN=Go Daddy Secure Certificate Authority - G2
*  SSL certificate verify ok.
* Using HTTP2, server supports multi-use
* Connection state changed (HTTP/2 confirmed)
* Copying HTTP/2 data in stream buffer to connection buffer after upgrade: len=0
* Using Stream ID: 1 (easy handle 0x55d5d0bd32f0)
> POST /api/incidents/6eddaf40-938c-11ed-b0f8-a7e340234a4e/mute?key=MbEoJwFYiPmMUDi8k1CaIIo3EDeFwWox HTTP/2
> Host: defender.lumu.io
> user-agent: curl/7.68.0
> accept: */*
> content-type: application/json
> content-length: 24
> 
* We are completely uploaded and fine
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* Connection state changed (MAX_CONCURRENT_STREAMS == 128)!
< HTTP/2 200 
< content-length: 0
< date: Mon, 06 Feb 2023 16:59:00 GMT
< x-ratelimit-remaining-minute: 49
< x-ratelimit-limit-minute: 50
< x-ratelimit-remaining-day: 9736
< x-ratelimit-limit-day: 10000
< ratelimit-reset: 60
< ratelimit-remaining: 49
< ratelimit-limit: 50
< trace-id: ac9142e0017c5f27
< x-kong-upstream-latency: 3
< x-kong-proxy-latency: 8
< via: kong/2.8.3, 1.1 76f9985b0c929a023355db8e462592f0.cloudfront.net (CloudFront)
< server: Lumu
< x-cache: Miss from cloudfront
< x-amz-cf-pop: MIA3-C2
< alt-svc: h3=":443"; ma=86400
< x-amz-cf-id: WK7yNAtaTOYU33nP-ob-E-ipAInhBy3P3IL7Ye5ITuH6IolzXK3dPg==
< x-xss-protection: 1
< x-frame-options: SAMEORIGIN
< referrer-policy: no-referrer-when-downgrade
< content-security-policy: default-src 'self'
< x-content-type-options: nosniff
< strict-transport-security: max-age=31536000; preload
< cache-control: no-cache, no-store, must-revalidate
< cross-origin-embedder-policy: unsafe-none
< cross-origin-opener-policy: same-origin-allow-popups
< cross-origin-resource-policy: cross-origin
< expires: 0
< permissions-policy: 
< pragma: no-cache
< x-permitted-cross-domain-policies: none
< 
* Connection #0 to host defender.lumu.io left intact
```

Reply:
thanks, implemented resp_type='text'